### PR TITLE
feat: Add php max_request_body_size

### PR DESCRIPTION
**Pull request by HazAT**

### CODE REVIEW

File: src/collections/_documentation/platforms/php/index.md [hunk 1]
```diff
@@ -269,6 +269,12 @@ The number of attempts that should be made to send an event before erroring
 and dropping it from the queue.
 By default this option is set to `6`.
 
+
+##### max_request_body_size
+
+This option represents the size limit in bytes beyond which the body of the request is not captured. Default value is `medium`. Possible values are `none` = never send body, `small`, `medium`, `alawys` = always send the body, no matter how large it is.
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/php/index.md]:
```suggestion
This option represents the size limit in bytes beyond which the body of the request is not captured. The default value is `medium`. Possible values are `none` = never send body, `small`, `medium`, `always` = always send the body, no matter how large it is.
```

File: src/collections/_documentation/platforms/php/index.md [hunk 2]
```diff
@@ -269,6 +269,12 @@ The number of attempts that should be made to send an event before erroring
...
+The values are chosen like this on purpose since the SDK internally can define what the exact limits are.
```
Comment (kamilogorek) [inline src/collections/_documentation/platforms/php/index.md]: Still not sure what those values mean. Maybe add some ranges? or current values?

Comment (bruno-garcia) [reply src/collections/_documentation/platforms/php/index.md]: I think this is an attempt to allow us to change this without 'breaking' customers.
Today you'd need to read the code to see what we consider `small`, `medium` etc.
i.e: 

https://github.com/getsentry/sentry-dotnet/blob/46bf5548d347ad1e206aefa3ea91cc46ab02004f/src/Sentry/Extensibility/RequestBodyExtractionDispatcher.cs#L45

File: src/collections/_documentation/platforms/php/index.md (hunk 1 above)
Comment (bruno-garcia) [reply src/collections/_documentation/platforms/php/index.md]: Isn't this part of the unified API?

File: src/collections/_documentation/platforms/php/index.md (hunk 2 above)
Comment (HazAT) [reply src/collections/_documentation/platforms/php/index.md]: ^ this

File: src/collections/_documentation/platforms/php/index.md (hunk 1 above)
Comment (HazAT) [reply src/collections/_documentation/platforms/php/index.md]: Yes, kinda, but as always we are only going to introduce this reactively to SDKs once ppl ask for it and not proactively push it out to all SDKs. 

For PHP it was there in the old SDK so we needed to bring it back.

**Comment by kamilogorek** [comment]
Merging so it won't get stuck.

**Pull request merged**